static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -111,14 +125,47 @@ export class ProteinProteinInteractionGraph extends React.Component<
     if (this.state.graphData === null) {
       return null;
     }
+    const data = this.state.graphData.linkDataNested
+      .slice(0, this.state.depth + 1)
+      .flat(1);
     return (
-      <>
-        <div id={GRAPH_ID} />
-        <ProteinProteinInteractionTable
-          data={this.state.graphData.linkDataNested
-            .slice(0, this.state.depth + 1)
-            .flat(1)}
-        />
+      <div className="ppi-container">
+        <Row className="justify-content-end">
+          <ButtonGroup>
+            <Button
+              className="btn btn-sm btn-light shadow-none"
+              onClick={() =>
+                this.setState({ showTableView: !this.state.showTableView })
+              }
+            >
+              <i className="material-icons align-middle">
+                {this.state.showTableView ? "hub" : "table_restaurant"}
+              </i>
+              <span>
+                {this.state.showTableView ? " Graph View" : " Table View"}
+              </span>
+            </Button>
+            <CSVLink
+              data={data}
```

Comment:
curious what the csv ends up looking like

Reply:
It looks like this:
![image](https://user-images.githubusercontent.com/27650355/183220481-506b70c7-703c-4cc5-8552-746dbffdd2ca.png)

One TODO is to make the score column header say "Confidence" instead.  Would you like me to address this now?

Reply:
is it an easy change to make the score column header say "Confidence" instead? If it's an easy change, let's do it now, if not, this is ok too

Reply:
yup just pushed something to make the column headers the same for the displayed table/downloaded table!

Reply:
The changes are that `CSVLInk` now takes a `headers` prop

```
 <CSVLink
              data={data}
              headers={COLUMNS.map(({ Header, accessor }) => ({
                label: Header,
                key: accessor,
              }))}
              filename={`${this.props.centerProteinDcid
                .replace("bio/", "")
                .toLowerCase()}_links.csv`}
              enclosingCharacter={""}
            >
```
 and that I now export `COLUMNS` form `protein_protein_interaction_table.tsx` and moved the type assertion into `Column<InteractionLink>[]` later

```
= useTable({
    columns: COLUMNS,
    data,
  } as { columns: Column<InteractionLink>[]; data: InteractionLink[] });
```

Reply:
nice! this looks good, feel free to merge!